geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreTest.scala

```diff
@@ -348,37 +347,26 @@ class AccumuloDataStoreTest extends Specification with TestWithMultipleSfts {
       val originalSchema = "name:String,dtg:Date,*geom:Point:srid=4326"
       val sftName = createNewSchema(originalSchema).getTypeName
 
-      "prevent changing default geometry" >> {
-        val modified =
-          SimpleFeatureTypes.createType(sftName, "name:String,dtg:Date,geom:Point:srid=4326,*geom2:Point:srid=4326")
-        modified.getUserData.putAll(ds.getSchema(sftName).getUserData)
-        ds.updateSchema(sftName, modified) should throwAn[UnsupportedOperationException]
-        val retrievedSchema = SimpleFeatureTypes.encodeType(ds.getSchema(sftName))
-        retrievedSchema mustEqual originalSchema
-      }
-      "prevent changing attribute order" >> {
-        val modified = SimpleFeatureTypes.createType(sftName, "dtg:Date,name:String,*geom:Point:srid=4326")
-        modified.getUserData.putAll(ds.getSchema(sftName).getUserData)
-        ds.updateSchema(sftName, modified) should throwA[UnsupportedOperationException]
-        val retrievedSchema = SimpleFeatureTypes.encodeType(ds.getSchema(sftName))
-        retrievedSchema mustEqual originalSchema
-      }
-      "prevent removing attributes" >> {
-        val modified = SimpleFeatureTypes.createType(sftName, "dtg:Date,*geom:Point:srid=4326")
-        modified.getUserData.putAll(ds.getSchema(sftName).getUserData)
-        ds.updateSchema(sftName, modified) should throwA[UnsupportedOperationException]
-        val retrievedSchema = SimpleFeatureTypes.encodeType(ds.getSchema(sftName))
-        retrievedSchema mustEqual originalSchema
-      }
-      "allow adding attributes" >> {
-        // note: we actually modify the schema here so this check is last
-        val newSchema = "name:String,dtg:Date,*geom:Point:srid=4326,newField:String"
-        val modified = SimpleFeatureTypes.createType(sftName, newSchema)
+      def modify(spec: String): Unit = {
+        val modified = SimpleFeatureTypes.createType(sftName, spec)
         modified.getUserData.putAll(ds.getSchema(sftName).getUserData)
         ds.updateSchema(sftName, modified)
-        val retrievedSchema = SimpleFeatureTypes.encodeType(ds.getSchema(sftName))
-        retrievedSchema mustEqual newSchema
       }
+
+      // "prevent changing default geometry" >> {
```

Comment:
What's the reason for combining these tests?

Reply:
making it so we can remove the `sequential` flag in the tests, which makes them run faster. it was only needed for this small block of tests so i just put them in one block so that they'd run sequentially but all the others could run in parallel